### DISCUSSION TITLE
Keine Funktion

### DISCUSSION THREAD

**Discussion by Manuel1607** [thkl/hap-homematic#333] [General]
Hallo, 

ich habe Homekit installiert und habe auch die Ports freigegebn, bzw alle Port sogar. Aber ich habe keinen Button unter Systemeinstellungen. Wenn ich auf Zusatzsoftware gehe und dann auf HomeKit, kommt das diese Seite nicht geladen werden konnte. 
Hat jemand eine Idee?

**Comment by gerd2005** [comment]
Hallo, schau mal hier:
#323

**Comment by Manuel1607** [comment]
Hallo, 

danke aber leider hilft es mir nicht, da die hier angesprochen Datein vorhanden sind. 


Wenn ich auf neustart oder deinstallierne gehe kommt immer eine Fehlermeldung. Kann mir jemand sagen, wo genau alles Dateien liegen dann lösche ich sie per FileZilla und probiere dann noch mal eine neuinstallation

**Comment by Manuel1607** [comment]
Nachdem ich bei den Sicherheitseinstellungen die Authentifizierung deaktiviert habe, kam ich drauf und konnte Einstellungen vornehmen, jedoch war dies nach dem nächsten Neustart wieder vorbei. Jemand noch eine idee?
